examples/inflate-helm-chart-remote-values-file/.expected/exec.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+kpt fn eval --image-pull-policy ifNotPresent --image gcr.io/kpt-fn/inflate-helm-chart:unstable \
+--network \
+--mount type=bind,src="$(pwd)",dst=/tmp/charts -- \
+name=cert-manager \
+namespace=cert-manager \
+releaseName=cert-manager \
+valuesFile=https://raw.githubusercontent.com/config-sync-examples/helm-components/main/cert-manager-values.yaml
```

Comment:
question: values.yaml can exist locally as well ?

Reply:
yes. The example `inflate-helm-chart-local` covers this